translations/en.json

```diff
@@ -718,6 +718,7 @@
 	"network": "Network",
 	"network-chain": "Network chain",
 	"network-details": "Network details",
+  "network-info": "Network info",
```

Comment:
I've noticed that en.json uses tabs instead of spaces. But does it even matter?

Reply:
yeah it does though if we could switch the whole file to what @rasom is using I'd prefer cause it's fucked up now I have to enable whitespace node and copy paste whitespaces to keep it even

Reply:
fixed